src/ocaml/driver/pparse.ml

```diff
@@ -138,28 +138,19 @@ let apply_rewriters ~ppx ?restore ~tool_name = function
   | `Implementation ast ->
     `Implementation (apply_rewriters_str ~ppx ?restore ~tool_name ast)
 
-(*
-let read_ast magic fn =
-  let ic = open_in_bin fn in
-  try
-    let buffer = really_input_string ic (String.length magic) in
-    assert(buffer = magic); (* already checked by apply_rewriter *)
-    Location.input_name := input_value ic;
-    let ast = input_value ic in
-    close_in ic;
-    Misc.remove_file fn;
-    ast
-  with exn ->
-    close_in ic;
-    Misc.remove_file fn;
-    raise exn
-
 let pp_commandline cmd fn_in fn_out =
   Printf.sprintf "%s %s 1>%s"
```

Comment:
Same kind of comment as in the other PR: we should either drop everything printed on `stderr`, or properly capture it.